persistence/sql/migrations/templates/20220301102702_identity_address_performance.up.sql

```diff
@@ -1,2 +0,0 @@
-UPDATE identity_recovery_addresses SET value = LOWER(value) WHERE TRUE;
```

Comment:
Could you re-add this file please? :)

Reply:
Do you mean to re-add the original version of the file referencing `identity_verification_addresses` or a modified one that refers to `identity_verifiable_addresses` and matches the migrations in `sql/`?

Reply:
Ah, I missed the typo! Please re-add the corrected version of this :)

Reply:
Thanks :) Restored the original template files and fixed the type in the table name.